vela-templates/definitions/internal/component/cron-task.cue

```diff
@@ -0,0 +1,275 @@
+"cron-task": {
+	type: "component"
+	annotations: {}
+	labels: {}
+	description: "Describes cron jobs that run code or a script to completion."
+	attributes: workload: {
+		definition: {
+			apiVersion: "batch/v1beta1"
+			kind:       "CronJob"
+		}
+		type: "cronjobs.batch"
+	}
+}
+template: {
+	output: {
+		apiVersion: "batch/v1beta1"
+		kind:       "CronJob"
+		spec: {
+			schedule:                   parameter.schedule
+			startingDeadlineSeconds:    parameter.startingDeadlineSeconds
+			concurrencyPolicy:          parameter.concurrencyPolicy
+			suspend:                    parameter.suspend
+			successfulJobsHistoryLimit: parameter.successfulJobsHistoryLimit
+			failedJobsHistoryLimit:     parameter.failedJobsHistoryLimit
+			jobTemplate: {
+				if parameter.labels != _|_ {
+					metadata: labels: parameter.labels
+				}
+				if parameter.annotations != _|_ {
+					metadata: annotations: parameter.annotations
+				}
+				spec: {
+					parallelism: parameter.count
```

Comment:
```suggestion
					parallelism: parameter.count
					ttlSecondsAfterFinished: parameter.ttl
					activeDeadlineSeconds: parameter.timeout
					backoffLimit: parameter.backoffLimit
```
1. plz add `ttlSecondsAfterFinished` support to avoid the shortage of IP resources in the k8s cluster due to the fact that the pods after the execution are not recycled in time.
2. add `activeDeadlineSeconds` support. 
3. export `backoffLimit` allow user to set the number of retries.

Reply:
ok, i'll add it